app/assets/javascripts/initializers/initializeVideoPlayback.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * This script hunts for video tags and initializes the correct player
+ * depending on the platform:
+ * - web: jwplayer
+ * - iOS/Android: Native player
+ *
+ * Once jwplayer is initialized there's no follow up actions to be taken.
+ * Mobile Native players send back information into the DOM in order to
+ * interact and update the UI, therefore a MutationObserver is registered.
+ */
+
+ /* eslint no-use-before-define: 0 */
+ /* eslint no-param-reassign: 0 */
+ /* eslint no-useless-escape: 0 */
+ /* global jwplayer */
+
+function initializeVideoPlayback() {
+  var nativeBridgeMessage;
+  var currentTime = "0";
+
+  function getById(name) {
+    return document.getElementById(name);
+  }
+
+  function isNativeIOS() {
+    return (
+      navigator.userAgent === 'DEV-Native-ios' &&
+      window &&
+      window.webkit &&
+      window.webkit.messageHandlers &&
+      window.webkit.messageHandlers.video
+    );
+  }
+
+  function isNativeAndroid() {
+    return (
+      navigator.userAgent === 'DEV-Native-android' &&
+      typeof AndroidBridge !== 'undefined' &&
+      AndroidBridge !== null &&
+      AndroidBridge.videoMessage !== undefined
+    );
+  }
+
+  function getParameterByName(name, url) {
+    if (!url) url = window.location.href;
+    name = name.replace(/[\[\]]/g, "\\$&");
```

Comment:
CodeQL is raising a warning here. This code was not modified from the current implementation (maybe wasn't caught before because it lived inline `_video_player.html.erb`).

Instead of fiddling with the regexp (and possibly breaking it) left it unchanged. If this is something we want to change I'd be happy to hear suggestions/comments on this

Reply:
Let's not touch it then. We also just removed CodeQL for the time being.